Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml

```diff
@@ -57,66 +57,66 @@
+    <constant name="FwdTracker_rmax2" value="1556*mm"/>
+    <constant name="FwdTracker_z2" value="3000*mm"/>
+    <constant name="FwdTracker_zOffset2" value="13000*mm"/>
+
+    <!-- CALORIMETER DIMENSIONS -->
+    <!-- ECal Barrel (EM B) -->
     <constant name="BarECal_id" value="5"/>
     <constant name="BarECal_rmin" value="1750*mm"/>
     <constant name="BarECal_rmax" value="2750*mm"/>
     <constant name="BarECal_dz" value="5000*mm"/>
-    <!-- E(M)Cal End-caps -->
-    <constant name="EndCapECal_id" value="6"/>
-    <constant name="EndCapECal_rmin1" value="864.278*mm"/>
-    <constant name="EndCapECal_rmax1" value="2800*mm"/>
-    <constant name="EndCapECal_rmin2" value="1467.641*mm"/>
-    <constant name="EndCapECal_rmax2" value="2800*mm"/>
-    <constant name="EndCapECal_dz" value="1850*mm"/>
-    <constant name="EndCapECal_zOffset" value="5300*mm + EndCapECal_dz"/>
-    <!-- E(M)Cal forward -->
-    <constant name="FwdECal_id" value="7"/>
-    <constant name="FwdECal_rmin" value="30*mm"/>
-    <constant name="FwdECal_rmax" value="3500*mm"/>
-    <constant name="FwdECal_dz" value="750*mm"/>
-    <constant name="FwdECal_zOffset" value="16500*mm+FwdECal_dz"/>
-
-
-    <!-- HCAL DIMENSIONS -->
-    <!-- HCal Barrel -->
+    <!-- HCal Barrel (HCAL B) -->
     <constant name="BarHCal_id" value="8"/>
     <constant name="BarHCal_rmin" value="2850*mm"/>
-    <constant name="BarHCal_rmax" value="4850*mm"/>
+    <constant name="BarHCal_rmax" value="4890*mm"/>
     <constant name="BarHCal_dz" value="4600*mm"/>
-    <!-- HCal End-Cap -->
-    <constant name="EndCapHCal_id" value="9"/>
-    <constant name="EndCapHCal_rmin" value="2850*mm"/>
-    <constant name="EndCapHCal_rmax" value="4550*mm"/>
-    <constant name="EndCapHCal_dz" value="2000*mm"/>
-    <constant name="EndCapHCal_zOffset" value="5000*mm + EndCapHCal_dz"/>
-    <!-- HCal forward -->
-    <constant name="FwdHCal_id" value="10"/>
-    <constant name="FwdHCal_rmin" value="30*mm"/>
-    <constant name="FwdHCal_rmax" value="3500*mm"/>
-    <constant name="FwdHCal_dz" value="1250*mm"/>
-    <constant name="FwdHCal_zOffset" value="18000*mm + FwdHCal_dz"/>
+    <!-- Extended barrel (EM EB + HEC)-->
+    <constant name="ExtBarECal_id" value="6"/>
+    <constant name="HadronicEndCap_id" value="7"/>
+    <constant name="ExtBarCal_rmin1" value="776*mm"/>
+    <constant name="ExtBarCal_rmax1" value="2700*mm"/>
+    <constant name="ExtBarCal_rmin2" value="1272*mm"/>
+    <constant name="ExtBarCal_rmax2" value="2700*mm"/>
+    <constant name="ExtBarCal_dz" value="1500*mm"/>
+    <constant name="ExtBarCal_zOffset" value="5300*mm + ExtBarCal_dz"/>
+    <!-- HCal extended barrel (HCAL EB) -->
+    <constant name="ExtBarHCal_id" value="9"/>
+    <constant name="ExtBarHCal_rmin1" value="3500*mm"/>
+    <constant name="ExtBarHCal_rmax1" value="4590*mm"/>
+    <constant name="ExtBarHCal_rmin2" value="2850*mm"/>
+    <constant name="ExtBarHCal_rmax2" value="4590*mm"/>
```

Comment:
I actually don't like it being called the same way (_rmin1 _rmax2 etc) as it is for Extended barrel above. The shapes are very different (trapezoid vs. two rectangles).
Wouldn't it be better called ExtBarHCal1_rmin or ExtBarHCalpart1_rmin etc. ? Same for dz below.